src/index.ts

```diff
@@ -571,7 +571,7 @@ prog
       _: string[]
     }) => {
       if (opts._.length === 0 && !opts['write-file']) {
-        const defaultInputs = ['src', 'test'].filter(fs.existsSync)
+        const defaultInputs = [].filter(fs.existsSync)
```

Comment:
Is `[].filter(...)` supposed to produce something other than a blank array?

Reply:
This was done to prevent the linter from linting all files when none changed. Have made this fix less of a hack in dd90c30888cf30ba57a2dc92c492cc7465268a5e